public-api/src/main/java/org/pmiops/workbench/publicapi/DataBrowserController.java

```diff
@@ -468,7 +468,6 @@ public ConceptAnalysis apply(ConceptAnalysis ca) {
             List<AchillesAnalysis> analyses = achillesAnalysisDao.findSurveyAnalysisResults(surveyConceptId, qlist);
             QuestionConcept.mapAnalysesToQuestions(questions, analyses);
         }
-
```

Comment:
nit: would probably revert all changes to this file since it's a whitespace-only change - it keep the per-file commit history/blame cleaner